english/forest/eat/eat.md

```diff
@@ -0,0 +1,3 @@
+In a panic, you stuff your face full of all the airplane pretzels and peanuts they gave you and slurp the last drop of the Coke. There were exactly 12 peanuts and 7 pretzels. You wonder if you should have rationed them out over the course of a week or so, one pretzel and two peanuts per day. But it is too late, and you're still hungry.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.